server/pulp/plugins/file/distributor.py

```diff
@@ -70,7 +70,7 @@ def publish_repo(self, repo, publish_conduit, config):
         :return:                report describing the publish operation
         :rtype:                 pulp.plugins.model.PublishReport
         """
-        if not config.get("skip_fast_forward", True):
+        if config.get("force_full", False):
```

Comment:
This doesn't make sense: force_full is supposed to *disable* fast-forward, not *enable* it.

- force_full=False should mean "don't force a full publish - it's OK to not process already-republished content"
- force_full=True should mean "force a full publish - process all content even if already published"

The behavior is now the wrong way around, where force_full *disables* fast-forward publish for rpm, rsync distributors, but *enables* it for file distributors.